shared/common-adapters/input.js.flow

```diff
@@ -1,14 +1,15 @@
 // @flow
 import {Component} from 'react'
+import type {Exact} from '../constants/types/more'
 
-export type Props = $Shape<{
+export type Props = Exact<{
```

Comment:
Using Exact here let's us limit to only the fields that are defined and nothing more. This in part helps catch cases where we do `onChange` instead of the optional `onChangeText`.